swagger.yaml

```diff
@@ -1,75 +1,83 @@
 swagger: '2.0'
+
 info:
```

Comment:
Could you put a link to the github repo somewhere up here? Like in externalDocs?